test/unit/minunit.h

```diff
@@ -168,15 +168,15 @@ void sprint_mem(char *out, const ut8 *buf, size_t len) {
 } while (0)
 
 #define mu_assert_nullable_streq(actual, expected, message) do { \
-		char _meqstr[2048]; \
+		char _meqstr[5120]; \
 		const char *act__ = (actual); \
 		const char *exp__ = (expected); \
 		sprintf(_meqstr, "%s: expected %s, got %s.", (message), (exp__ ? exp__ : "NULL"), (act__ ? act__ : "NULL")); \
```

Comment:
how about converting this `sprintf` to `snprintf`?

Reply:
i'll do that in a different pr